assets/chat/js/regex.js

```diff
@@ -1,3 +1,21 @@
+const regexslashcmd = /^\/([a-z0-9]+)[\s]?/i;
+const regextime = /(\d+(?:\.\d*)?)([a-z]+)?/gi;
```

Comment:
I wasn't even aware there's support for decimal notation here. I think we can remove this to simplify things. I'm 100% positive nobody uses it.
```suggestion
const regextime = /(\d+)([a-z]+)?/gi;
```

There's also the issue of commands that accept a time string succeeding when it's incorrectly formatted, but that's something we'll address in a future PR.